### ISSUE TITLE
Make html - Error with import uwsgi

### ISSUE THREAD

**Issue by jdugh** [onaio/onadata#1395]
I try to install onadata with this installation guide [https://api.ona.io/static/docs/install.html](https://api.ona.io/static/docs/install.html).
All is OK, but when I try to `make html` I have this error :
`/home/ona/onadata/docs/index.rst:70: WARNING: toctree contains reference to nonexisting document u'  :maxdepth: 2'
WARNING: /home/ona/onadata/docs/onadata.rst:27: (WARNING/2) autodoc: failed to import module u'onadata.devwsgi'; the following exception was raised:
Traceback (most recent call last):
  File "/home/ona/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 657, in import_object
    __import__(self.modname)
  File "/home/ona/onadata/onadata/devwsgi.py", line 11, in <module>
    import uwsgi
ImportError: No module named uwsgi`
The full stack trace is extermely long !

`$pip freeze | grep uWSGI`
`uWSGI==2.0.15`

Ubuntu 16.04

Do you have an idea where my problem comes from?

Aha! Link: https://ona.aha.io/features/PROD-355

**Comment by ukanga** [comment]
I have tested this in a docker container and it seems to be working as expected. Are you using a virtual environment? The Dockerfile is https://github.com/onaio/onadata/blob/master/Dockerfile.

**Comment by jdugh** [comment]
@ukanga  Yes, I use a virtual env (I try to install without Docker).
I clone the repo https://github.com/onaio/onadata.git
I run [ubuntu script install](https://api.ona.io/static/docs/install.html#prepare-os)
I run [psql command](https://api.ona.io/static/docs/install.html#in-the-base-os)
I set and run [my virtualenv](https://api.ona.io/static/docs/install.html#set-up-and-start-your-virtual-environment-or-sandbox)
I  [create my local_settings.py](https://api.ona.io/static/docs/install.html#create-a-local-settings-py-and-update-it-accordingly)
I run [make](https://api.ona.io/static/docs/install.html#run-make-to-set-up-onadata-and-for-initial-db-setup) : All is OK
And i try to  [compile docs](https://api.ona.io/static/docs/install.html#compile-api-docs)
`cd docs`
`make html`
And I have the previous error

**Comment by ukanga** [comment]
@jdugh I cannot seem to reproduce this error. I will suggest you start the instructions again. I will also recommend you use a virtual environment. I have reordered the instructions in this branch https://github.com/onaio/onadata/blob/update-install-instructions, see https://github.com/onaio/onadata/blob/update-install-instructions/docs/install.rst. I will also suggest you use this branch which is still work in progress with regards to the documentation to do your setup. I have made some changes to the install script and `Makefile`, see https://github.com/onaio/onadata/commit/d1adc0a4b7cef2faa932c8209b249bd583e3ef7e.

**Comment by jdugh** [comment]
@ukanga Thanks for updating the install instructions.
Unfortunately, I have the same warning/error. I have start a new install with this instructions, I used à virtual environnement as indicated and I used your commit. I test with the latest master branch too.
I link a file with full stack trace, maybe their are more info
[stack_onadata.txt](https://github.com/onaio/onadata/files/2005052/stack_onadata.txt)
